recipes/blas_openblas/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: blas_openblas
+  version: 1.0.0
+
+build:
+  number: 0
+  track_features:
+    - blas_openblas
+
+test:
+  commands:
+    - conda list blas_openblas
```

Comment:
Opened this issue ( https://github.com/conda/conda-build/issues/910 ) with regards to this error.